bittide-shake/exe/Main.hs

```diff
@@ -224,6 +225,15 @@ targets =
         { targetPostProcess = Just "post-vex-riscv-test"
         , targetExtraXdc = ["jtag_config.xdc", "jtag_pmod1.xdc"]
         }
+    , (testTarget "Bittide.Instances.Hitl.Ethernet.vexRiscvTcpTest")
+        { targetPostProcess = Just "post-vex-riscv-tcp-test"
+        , targetHasXdc = True
+        , targetExternalHdl =
+            [ "$env(VERILOG_ETHERNET_SRC)/rtl/*.v"
+            , "$env(VERILOG_ETHERNET_SRC)/lib/axis/rtl/*.v"
```

Comment:
Could you make an issue saying we should remove this in favor of some Haskell code finding these files? Splicing in arbitrary TCL feels like a hell of a hack.

Reply:
https://github.com/bittide/bittide-hardware/issues/622